.github/CODEOWNERS

```
@@ -52,6 +52,7 @@ packages/kbn-babel-register @elastic/kibana-operations
 packages/kbn-babel-transform @elastic/kibana-operations
 x-pack/plugins/banners @elastic/appex-sharedux
 packages/kbn-bazel-runner @elastic/kibana-operations
+packages/kbn-bfetch-error @elastic/kibana-data-discovery
```

```suggestion
packages/kbn-bfetch-error @elastic/appex-sharedux
```
Should @elastic/appex-sharedux own this since they own bfetch?

---

Looks like the change here got auto-reverted in https://github.com/elastic/kibana/pull/171563/commits/ba7b05d3011ea40117e6003e38c9a463c0d9d2c2. I'm guessing I should have asked for this change in `packages/kbn-bfetch-error/kibana.jsonc` instead (oops).